### ISSUE TITLE
ds34vita diagnols still not working, but found alternative

### ISSUE THREAD

**Issue by kanjieater** [MERLev/reVita#57]
Noticed in muramasa the roll command doesn't work due to diagnol analog movements not working. This version works, but sadly no ds3 support. Wondering if this could be added to ds34vita?
https://github.com/gimmer/ds4vita/releases/tag/diagonals1.1

**Comment by admo64** [comment]
Hello, Hassman0 fork have a release having yzzyx-network changes and increased system boot speed

https://github.com/Haasman0/reVita/releases/tag/Release

This fork have Analog Wide Mode option restored, and disabling it (is on by default) fixed my diagonal issue with Sly Cooper 4 using the Vita's analogs. You could test if it works in DS3 or DS4

IMPORTANT: For the change to be effective you have to close and reopen the game after saving the profile in reVita

**Comment by kanjieater** [comment]
yeah i'm using minivitatv with the wiki's plugin settings, and the fork you linked. seems like diagnols are working in muramasa. Thanks
